flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
+}
```

Comment:
What does `telegram_web.png` do here?

Reply:
I plan on adding these images to the metainfo after this gets its repo